src/test/java/SearchServletTest.java

```diff
@@ -276,6 +290,32 @@ public void checkParseExceptionIsThrown() throws IOException {
     verify(response).setStatus(HttpServletResponse.SC_BAD_REQUEST);
   }
 
+  @Test
+  public void queryReceiptsWithDifferentUserId() throws IOException {
+    // All receipts under the userId: "testId".
+    //
+    // id   Timestamp      Price          Store                    Categories
+    // 1  1045237591000    26.12        "walmart"         ["candy", "drink", "personal"]
+    // 2  1560193140000    14.51        "contoso"         ["cappuccino", "sandwich", "lunch"]
+    // 3  1491582960000    29.01   "main st restaurant"   ["food", "meal", "lunch"]
+
+    // Add mock receipts to datastore.
+    ImmutableSet<Entity> expectedReceipts = TestUtils.addTestReceipts(datastore);
+
+    // Set userId to one that doesn't have any receipts under it.
+    helper.setEnvAttributes(new HashMap(
+        ImmutableMap.of("com.google.appengine.api.users.UserService.user_id_key", "wrongUserId")));
+    helper.setUp();
```

Comment:
The `.setup()` can be added to the end of the same line.

https://cloud.google.com/appengine/docs/standard/java/tools/localunittesting/javadoc/com/google/appengine/tools/development/testing/LocalServiceTestHelper.html#setEnvAttributes-java.util.Map-